### PR TITLE
Update and extend the contributing guidelines for tutorials

### LINKED ISSUE #164: Missing contribution guidelines
We currently have some general information on contributing (tutorials) on the [page "Contribute to preCICE"](https://www.precice.org/community-contribute-to-precice.html). We should add a `CONTRIBUTING.md` file with more details, where we could also inform the contributors about our workflows (and checks such as #163).

### PR DESCRIPTION
**Pull request by MakisH**
Closes https://github.com/precice/tutorials/issues/164

@uekerman do we need to add anything else in this page?

### CODE REVIEW

File: pages/community/community-contribute-to-precice.md [hunk 1]
```diff
@@ -52,23 +52,63 @@ You can also use preCICE from the [develop branch](https://github.com/precice/pr
...
+- Data names are in singular, e.g. `Stress`, `Heat-Flux`.
+- Mesh names start with the participant/domain name, e.g. `Fluid-Mesh`.
+- Watchpoint names should be describing the point, not be a generic name. 
+
+### What to check before submitting
+- Check your shell scripts with [shellcheck](https://github.com/koalaman/shellcheck/):
+   ```bash
+   shellcheck <script.sh>
+   ```
+   and format them with any formatted (e.g. make sure there is an empty line at the end of the script).
```
Comment (uekerman) [inline pages/community/community-contribute-to-precice.md]:
```suggestion
   and format them with any format (e.g. make sure there is an empty line at the end of the script).
```
?

**Review by uekerman — APPROVED** [review]
I like it. 
Some comments:
* We could say what we expect people to write in the PR description. Maybe a screenshot of the simulation would be helpful there.
* And they should say which versions (solvers, adapters, preCICE) they used.
* If non-official solvers / adapters are used, we need some hints in the README how to get those.

File: pages/community/community-contribute-to-precice.md (hunk 1 above)
Comment (MakisH) [reply pages/community/community-contribute-to-precice.md]: Should be `formatter`, sorry (I tend to do this kind of typo very often...). Fixed.

**Comment by MakisH** [comment]
I added a few notes on the PR description. Let me know if we need anything else, otherwise I will merge in the next hour.

**Review by uekerman — APPROVED** [review]

**Pull request merged**